test-requirements.txt

```diff
@@ -4,7 +4,7 @@ pytest-cov>=1.8.1
 pytest-timeout>=1.0.0
 unittest2>=1.1.0
 grpcio-tools>=1.0.0
-google-auth>=0.2.0
+google-auth>=0.5.0
```

Comment:
Yes, let's revert the changes made when working on the Travis error unless they're related to the rest of this PR.